plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -833,4 +863,18 @@ SecRule REQUEST_FILENAME "@contains /apps/notes/notes/" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.title,\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
 
+#
+# [ Custom CSS ]
+#
+
+# Allow admin to add CSS custom CSS code
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/provisioning_api/api/v[0-9]/config/apps/theming_customcss/customcss$"
```

Comment:
Missing ` \` at the end of line.

Reply:
the rule will not work if I add ``/`` at the end of the ``REQUEST_FILENAME``, that's how the request is made.